serving/src/test/resources/docker-compose/docker-compose-it-core.yml

```diff
@@ -0,0 +1,53 @@
+version: '3'
```

Comment:
Would it be possible for us to use the docker-compose file in `infra/docker-compose`? Is there anything set differently here?

Reply:
Only change is, this doesn't contain jupyter container. Also there was an issue with the context(relative path) for `DockerComposeContainer`, when you have multiple docker-compose files. I could define multiple `DockerComposeContainer` but that increased the boot time and also adds an extra container(I guess for management/networking) for each DockerComposeContainer.